docs/research-software/openfoam.md

```diff
@@ -79,6 +79,31 @@ Note there are no default module versions specified. It is recommended to
 use a fully qualified module name (with the exact version, as in the
 example above).
 
+### Extensions to OpenFOAM
+
+Many packages extend the central OpenFOAM functionality in some way. However,
+there is no completely standardised way in which this works. Some packages
+assume they have write access to the main OpenFOAM installation. If this is
+the case, you must install your own version before continuing. This
+can be done on an individual basis, or a per-project basis using the
+project shared directories.
```

Comment:
Suggestion: could you add this link to the "project shared directories" text?
https://docs.archer2.ac.uk/user-guide/data/#sharing-data-with-archer2-users-in-your-project